torch/distributed/nn/api/remote_module.py

```diff
@@ -96,6 +110,7 @@ def __init__(
                 >>> module_cls = MyModule
             args (Sequence, optional): args to be passed to ``module_cls``.
             kwargs (Dict, optional): kwargs to be passed to ``module_cls``.
+            device (torch.device): Device on the destination worker where we‘d like to place this module.
```

Comment:
Forgot to mention this earlier, but it might be good to make `device` the second parameter (right after we specify the destination). So invocations would be like `RemoteModule("trainer0", "cuda:3", ...)`. We can probably make this API change since the feature hasn't been announced yet.

Reply:
The only drawback here is the backward compatibility, because the previous second parameter is module_cls, which does not have a default value. Now this means device cannot have a default value.

This should be fine, since the constructor has only been used in a few unit tests.